philips_hue.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import requests
+import json
+from dotenv import load_dotenv
+
+load_dotenv()
+URL = os.environ.get("philips-url")
```

Comment:
Yeah I mostly did this as a temporary way to load sensitive information like the Philips Hue API key and IP address (which I stored in a .env file locally). I was planning on having this replaced once I have access to the Google Cloud Secret Manager. Would you still recommend for now moving this to a more centralized place?

Reply:
So I think that this actually supports the idea of making config loading more generalizable? Since already, we need a way to load config variables from multiple sources without modifying the philips_hue.py code (which has nothing to do with app configuration)?

But I think that it's okay to iterate (e.g. either postpone config loading to a later PR, or submit a separate config loading PR first and then come back to this). However, I think that the idea is that the client library code shouldn't have to change just because the app consumes configs from a different data source.

Reply:
That makes sense. I'll work on this in a separate branch/PR.